### PR TITLE
add covid-related agreements

### PR DESCRIPTION
**Pull request by tingshanL**
I added the agreements about covid we discussed on Sep over lab meeting.

### CODE REVIEW

**Review by jshinm — COMMENTED** [review]
Thank you, @tingshanL! 

While looking great, I made minor changes.

**Review by jshinm — DISMISSED** [review]

**Comment by jshinm** [comment]
@tathey1 anything we should change here?

File: content/about/agreements.md [hunk 1]
```diff
@@ -279,7 +279,14 @@ We collectively co-create our community.  Being a part of this community affords
 ### Jovo specific responsibilities
 
 - I am responsible for providing the financial, computational, space, and other resources appropriate for your position, potentially including salary, benefits,  computing costs, conference fees, travel expenses, publication fees, a desk, a laptop, monitors, etc.  Crucially, expenses never impact your salary or benefits. 
-- I am rensponsible for devoting at least 1 hour per week in person with each PhD student, except for weeks that I am traveling, or when the week is short due to academic calendar days off, or in the case of a disease/family emergency. These meetings will always be in person unless Jovo is not in person for some other commitment to family. 
+- I am responsible for devoting at least 1 hour per week in person with each PhD student, except for weeks that I am traveling, or when the week is short due to academic calendar days off, or in the case of a disease/family emergency. These meetings will always be in person unless Jovo is not in person for some other commitment to family.
+
+### COVID-19 related responsibilities
+
+If you test positive and/or have systems and/or were exposed to COVID-19,
```
Comment (tathey1) [inline content/about/agreements.md]: symptoms not systems (typo)

**Review by tathey1 — DISMISSED** [review]

File: content/about/agreements.md (hunk 1 above)
Comment (jshinm) [reply content/about/agreements.md]: fixed the typo and merging this PR. Thank you @tathey1 and @tliu68!

**Pull request merged**